lang/de.json

```diff
@@ -291,6 +291,7 @@
   "ConnectPaypal.processing": "In Bearbeitung...",
   "ConnectPaypal.refill": "Refill balance",
   "Contact": "Kontakt",
+  "contribute.changeFrequency.confirmMsg": "If you're changing the frequency, we will not contribute to this specific tier 'Gold Sponsor'",
```

Comment:
Translation files are still holding the old string with 'Gold Sponsor', `npm run langs:update to the rescue?`

Reply:
I ran this but it did not update the translation files. I had to change it manually

Reply:
@sudo-kaizen I think the reason for it may be that you're using Windows, were new lines are represented by a `\r\n`. The script on CI runs on a Linux, which uses a standard `\n`. Can you check your editor settings to see if there's an option for that and update the default message accordingly?

@znarf I'm not sure if https://eslint.org/docs/rules/linebreak-style is enabled, maybe it can help?

Reply:
@Betree I think we're talking about different things, my comment here was about the hardcoded "Gold Sponsor" in "contribute.changeFrequency.confirmMsg"string still appearing in translation files.

Reply:
Does this have anything to do with CRLF and LF? My editor is set to CRLF

Reply:
About the hardcoded "Gold Sponsor" in "contribute.changeFrequency.confirmMsg"string still appearing in translation files, that has been fixed. 
The test is failing at something else, and it is related to translations